docs/kubectl.md

```diff
@@ -450,9 +450,9 @@ Usage:
 
```

Comment:
Line [449](https://github.com/GoogleCloudPlatform/kubernetes/pull/3768/files#diff-2cd920f71449fac2cbb59c9daecd0aa4R449) is the one that needs to be changed. Usage string should list server|insecure-skip-tls-verify|certificate-authority|api-version as flags.